tests/integration/nupic/opf/opf_checkpoint_test/opf_checkpoint_test.py

```diff
@@ -25,7 +25,7 @@
 import shutil
 
 from nupic.data.file_record_stream import FileRecordStream
-from nupic.frameworks.opf.experiment_runner import runExperiment
+from nupic.frameworks.opf.experiment_runner import runExperiment, _getCheckpointParentDir
```

Comment:
Can we change `_getCheckpointParentDir` to be a public function explicitly? It seems like something that clients of experiment runner may wish to use.